srcpkgs/llvm14/template

```diff
@@ -168,6 +196,9 @@ pre_configure() {
 		cd ../..
 	fi
 
+	mkdir -p gnu
```

Comment:
With `-DCOMPILER_RT_DEFAULT_TARGET_ONLY=YES` I believe you don't need this. In any case having to touch empty files seems wrong anyway.